tests/FunctionalJavascript/AddCorporateFormTest.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_contact_forms\FunctionalJavascript;
+
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+
+/**
+ * Test Thirdparty settings on contact forms.
+ */
+class AddCorporateFormTest extends WebDriverTestBase {
+
+  /**
+   * An test user with permission to create contact forms.
+   *
+   * @var \Drupal\user\UserInterface
+   */
+  protected $testuser;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'user',
+    'oe_contact_forms',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    // Create and login test user.
+    $this->testuser = $this->drupalCreateUser([
+      'administer contact forms',
+    ]);
+    $this->drupalLogin($this->testuser);
+  }
+
+  /**
+   * Tests the corporate field requirements.
+   *
+   * First we test requirements for state dependent fields,
```

Comment:
These comments are not needed. Rather include at the right places in the code you test in.

Reply:
Fixed